.rubocop.yml

```
@@ -0,0 +1,104 @@
+# Configuration hierarchy:
+#
+# 1. Rubocop defaults
+# 2. Carbon Five defaults (this file)
+# 3. Project overrides
+#
+# See http://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-remote-url for details.
+#
+
+AllCops:
+  DisplayCopNames: true
+  DisplayStyleGuide: true
```

Non-blocking: It would be good to explicitly specify the version of Ruby compatibility that we are aiming for here:

```suggestion
  DisplayStyleGuide: true
  TargetRubyVersion: 2.4
```

We can do this in a separate PR when we fix the other non-autocorrectable RuboCop issues.

---

Interestingly, rubocop complains that the gemspec required version doesn't match that of the rubocop.yml so fixing this will fix a rubocop warning.